_component_design/lists.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Lists
+parent: Content Areas
+layout: component
+category: Components
+usage: >
+  Displays a grouped list of unordered or ordered objects within a UI.
```

Comment:
Are items in a list "objects" or "items"? If we user the word "items" in the content, let's change this to use "items" instead of "objects".

Reply:
Updated, also:

@DonNateR Feel free to write / rework the usage section fully. I wrote this as placeholder.

Reply:
@DonNateR Nm, I updated it to, " Lists help break up large blocks of text, make complex articles and blog posts easier to grasp, and make key information stand out."